libvirt/tests/src/migration/migration_misc/qemu_binary_change.py

```
@@ -0,0 +1,53 @@
+
+    """
+    def setup_test():
+        """
+        Setup steps
+
+        """
+        test.log.info("Setup steps for cases.")
+        migration_obj.setup_connection()
+        cmd = ("nohup bash -c 'while true; do chmod g+w /usr/libexec/qemu-kvm; "
+               "chmod g-w /usr/libexec/qemu-kvm; sleep 0.1; done' > /dev/null 2>&1 &")
+        process.run(cmd, shell=True)
```

Ruff (0.14.2)

Function call with `shell=True` parameter identified, security issue

(S604)